src/main/java/genesis/Genesis.java

```diff
@@ -0,0 +1,530 @@
+package genesis;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
```

Comment:
Should the java.io imports be placed together to keep the format of import order consistent?